.github/workflows/stale.yml

```diff
@@ -17,7 +17,7 @@ on:
   schedule:
   - cron: "30 1 * * *"
 
-permissions: read-all
+permissions: 
```

Comment:
Since no permissions are declared, there's nothing the action can do, e.g., label the issue, etc. So what does the action do?

Reply:
The specific job has permissions.
```
permissions:
      issues: write
      pull-requests: write
```